### PR TITLE
android: smaller new chat button in 1-hand UI mode, line on the correct side of the bar

### PR DESCRIPTION
**Pull request by epoberezkin**
- [x] scroll title together with the view
- [x] paddings in new chat view
- [x] line between toolbar and search
    - [x] chat list
    - [x] chat view
- [x]  not closing search when it's focussed and keyboard is open
- [x] flip animation direction for user picker
- [x]  let's try the same size for new chat button as circle avatar (possibly, without reducing the pencil size) - it looks too big now
- [x] desktop: add contact / scan,paste -> info and incognito info page - should be in left modal
- [x] accepting contact request with sndReady contact should open chat when accepted in chat list same as on new chat sheet?
- [x] Hide toolbar on one hand UI when searching
- [x] One hand UI modal toolbar have extra padding

### CODE REVIEW

**Review by diogofcunha — DISMISSED** [review]

File: apps/multiplatform/common/src/commonMain/kotlin/chat/simplex/common/views/chatlist/ChatListView.kt [hunk 1]
```diff
@@ -88,19 +86,21 @@ fun ChatListView(chatModel: ChatModel, settingsState: SettingsViewState, setPerf
...
+          Divider()
         }
       }
     },
     bottomBar = {
       if (oneHandUI.state.value) {
-        Box(Modifier.padding(end = endPadding)) {
+        Column(Modifier.padding(end = endPadding)) {
+          Divider()
           ChatListToolbar(
```
Comment (avently) [inline apps/multiplatform/common/src/commonMain/kotlin/chat/simplex/common/views/chatlist/ChatListView.kt:104]: There is a 1.dp difference in chat list bottom toolbar and new chat sheet bottom toolbar (new chat sheet is smaller in height)

File: apps/multiplatform/common/src/commonMain/kotlin/chat/simplex/common/views/chatlist/UserPicker.kt [hunk 2]
```diff
@@ -149,7 +149,7 @@ fun UserPicker(
     .padding(bottom = 10.dp, top = 10.dp)
     .graphicsLayer {
       alpha = animatedFloat.value
-      translationY = (animatedFloat.value - 1) * xOffset
+      translationY = (if (chatModel.controller.appPrefs.oneHandUI.state.value) -1 else 1) * (animatedFloat.value - 1) * xOffset
```
Comment (avently) [inline apps/multiplatform/common/src/commonMain/kotlin/chat/simplex/common/views/chatlist/UserPicker.kt:152]: you can just use this way to get appPrefs:
```kotlin
appPrefs.oneHandUI.....
```

No need to specify chatModel.controller

File: apps/multiplatform/common/src/commonMain/kotlin/chat/simplex/common/views/newchat/NewChatSheet.kt [hunk 3]
```diff
@@ -40,44 +40,54 @@ import java.net.URI
...
-    modifier = Modifier.fillMaxSize()
-  ) {
-    if (!oneHandUI.state.value) {
-      Box(contentAlignment = Alignment.Center) {
-        val bottomPadding = DEFAULT_PADDING
-        AppBarTitle(
-          stringResource(MR.strings.new_chat),
-          hostDevice(rh?.remoteHostId),
-          bottomPadding = bottomPadding
+  LaunchedEffect(keyboardState, oneHandUI) {
```
Comment (avently) [inline apps/multiplatform/common/src/commonMain/kotlin/chat/simplex/common/views/newchat/NewChatSheet.kt:46]: You are doing something strange here. On line 43 you remember SharedPreferences which will never be changed. Intead, you need to remember state like so:
```kotlin
  val oneHandUI = remember { chatModel.controller.appPrefs.oneHandUI.state }

```
And write in LaunchedEffect like so:

```kotlin
  LaunchedEffect(keyboardState, oneHandUI.value) {

```

File: apps/multiplatform/common/src/commonMain/kotlin/chat/simplex/common/views/newchat/NewChatSheet.kt [hunk 4]
```diff
@@ -40,44 +40,54 @@ import java.net.URI
...
 
-    val closeAll = { ModalManager.start.closeModals() }
-
-    var modifier = Modifier.fillMaxSize()
+      var modifier = Modifier.fillMaxSize()
 
-    if (oneHandUI.state.value) {
-      modifier = modifier.scale(scaleX = 1f, scaleY = -1f)
-    }
+      if (oneHandUI.state.value) {
```
Comment (avently) [inline apps/multiplatform/common/src/commonMain/kotlin/chat/simplex/common/views/newchat/NewChatSheet.kt:72]: why not just:

```kotlin 
Column(modifier = if (oneHandUI.value) Modifier.fillMaxSize().scale(scaleX = 1f, scaleY = -1f) else Modifier.fillMaxSize()
```

File: apps/multiplatform/common/src/commonMain/kotlin/chat/simplex/common/views/newchat/NewChatSheet.kt [hunk 5]
```diff
@@ -116,8 +126,8 @@ private fun NewChatSheetLayout(
   scanPaste: () -> Unit,
   createGroup: () -> Unit,
   close: () -> Unit,
-  oneHandUI: SharedPreference<Boolean>
 ) {
+  val oneHandUI = remember { chatModel.controller.appPrefs.oneHandUI }
```
Comment (avently) [inline apps/multiplatform/common/src/commonMain/kotlin/chat/simplex/common/views/newchat/NewChatSheet.kt:130]: same as before

File: apps/multiplatform/common/src/commonMain/kotlin/chat/simplex/common/views/newchat/NewChatSheet.kt [hunk 6]
```diff
@@ -326,8 +347,9 @@ private fun ContactsSearchBar(
   searchShowingSimplexLink: MutableState<Boolean>,
   searchChatFilteredBySimplexLink: MutableState<String?>,
   close: () -> Unit,
-  oneHandUI: SharedPreference<Boolean>
 ) {
+  val oneHandUI = remember { chatModel.controller.appPrefs.oneHandUI }
```
Comment (avently) [inline apps/multiplatform/common/src/commonMain/kotlin/chat/simplex/common/views/newchat/NewChatSheet.kt:351]: same

File: apps/multiplatform/common/src/commonMain/kotlin/chat/simplex/common/views/newchat/NewChatSheet.kt [hunk 7]
```diff
@@ -512,82 +534,105 @@ private fun contactTypesSearchTargets(baseContactTypes: List<ContactType>, searc
...
-    val listState = rememberLazyListState(lazyListState.first, lazyListState.second)
-    val searchText = rememberSaveable(stateSaver = TextFieldValue.Saver) { mutableStateOf(TextFieldValue("")) }
-    val searchShowingSimplexLink = remember { mutableStateOf(false) }
-    val searchChatFilteredBySimplexLink = remember { mutableStateOf<String?>(null) }
-    val showUnreadAndFavorites = remember { ChatController.appPrefs.showUnreadAndFavorites.state }.value
-    val contactTypes = listOf(ContactType.CHAT_DELETED)
-    val allChats by remember(chatModel.chats.value, contactTypes) {
-      derivedStateOf { filterContactTypes(chatModel.chats.value, contactTypes) }
+    if (oneHandUI.state.value) {
+      modifier = modifier.scale(scaleX = 1f, scaleY = -1f)
```
Comment (avently) [inline apps/multiplatform/common/src/commonMain/kotlin/chat/simplex/common/views/newchat/NewChatSheet.kt:564]: same

**Pull request merged**